src/main/java/org/example/models/JobRoleRequest.java

```diff
@@ -0,0 +1,104 @@
+package org.example.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.sql.Date;
+
+
+public class JobRoleRequest {
+
+    private String roleName;
+    private int location;
+    private int capability;
+    private int band;
+    private Date closingDate;
+    private String description;
+    private String responsibilities;
+    private String link;
+
+    @JsonCreator
+    public JobRoleRequest(@JsonProperty("roleName") final String roleName,
```

Comment:
create role ticket also asks for number of open positions

Reply:
This will be added with the addRole ticket